packages/admin/dashboard/src/routes/reservations/reservation-detail/reservation-detail.tsx

```diff
@@ -23,6 +24,9 @@ export const ReservationDetail = () => {
     }
   )
 
+  // TEMP: fetch directly since the fields are not populated with reservation call
```

Comment:
q: would it be more correct to fetch the item details?

Reply:
what endpoint would you suggest, this is fetching inventory item

Reply:
I am asking wrt. the comment you put here - what would be the optimal approach to make this not `TEMP`? Is it just adding the `inventory_item` field when fetching the reservation?

If so we should maybe either include that in this PR or create a separate task so that can be fixed

Reply:
> Is it just adding the inventory_item field when fetching the reservation?

Yup, ideally we would want that but with this approach computed properties such as `reserved_quantity` are not being returned. I'll open a ticket.